react/features/large-video/components/LargeVideo.web.js

```diff
@@ -70,6 +72,8 @@ export default class LargeVideo extends Component<*> {
                             muted = { true } />
                     </div>
                 </div>
+                { config.disableTranscriptionSubtitles
```

Comment:
should we be consistent with recording, which has `recordingEnabled: False`?

Reply:
I'm surprised you found any consistency to latch onto. I'm wondering more if subtitles should be enabled by default or disabled by default. Currently a missing `config.disableTranscriptionSubtitles` value means subtitles will be enabled by default.

Reply:
What about users being able to choose whether they want subtitles, which will be true by default so they can discover the feature and turn it off when necessary? While we don't have this yet, we will keep it disabled by default because it might not be used very often?

Reply:
Nik and I cleared this up. I wanted to confirm the default should be "on" and that is what it should be.